tests/rptest/clients/rpk.py

```diff
@@ -88,6 +88,20 @@ def partition_line(line):
 
         return filter(lambda p: p != None, map(partition_line, lines))
 
+    def wasm_deploy(self, script, description):
+        cmd = [
+            self._rpk_binary(), 'wasm', 'deploy', script, '--brokers',
+            self._redpanda.brokers(1), '--description', description
```

Comment:
I just realized we should randomize these commands to use any of the available brokers. What do you think?

Reply:
I think we can pass all brokers, would that suffice?